### PR TITLE
Update demo

### PR DESCRIPTION
**Pull request by woprandi**
I updated one of the demo to add a button to show/hide a table column. I also added a small example in the Filtered class

### CODE REVIEW

File: framework/source/class/qx/ui/table/model/Filtered.js [hunk 1]
```diff
@@ -20,6 +20,51 @@
...
+}, {
+  login : "luke",
+  name : "Luke Skywalker",
+  email : "luke@tatooine.org"
+}, {
+  login : "obi-wan",
+  name : "Obi-Wan Kenobi",
+  email : "obiwan@jedi.org"
+}];
+
```
Comment (oetiker) [inline framework/source/class/qx/ui/table/model/Filtered.js]: can't we have the address of Rey too ?

**Comment by woprandi** [comment]
@oetiker just for you ;)

File: framework/source/class/qx/ui/table/model/Filtered.js [hunk 2]
```diff
@@ -20,6 +20,55 @@
...
+  name : "Luke Skywalker",
+  email : "luke@tatooine.org"
+}, {
+  login : "obi-wan",
+  name : "Obi-Wan Kenobi",
+  email : "obiwan@jedi.org"
+}, {
+  login : "rey",
+  name : "Rey",
+  email : rey@jakku.sw
```
Comment (oetiker) [inline framework/source/class/qx/ui/table/model/Filtered.js]: quotes are missing

**Comment by zaucker** [comment]
Wouldn't it make sense  to call the button variable "checkbox" instead? It is a checkbox and not a button after all ...

**Comment by woprandi** [comment]
@zaucker You're right ;)

**Comment by level420** [comment]
five votes!
merging
Thank you @woprandi

**Pull request merged**